sdkproject/Assets/Mapbox/Unity/Constants.cs

```diff
@@ -28,8 +39,49 @@ public static class Math
 			public static readonly Vector3 Vector3Down = new Vector3(0, -1, 0);
 			public static readonly Vector3 Vector3One = new Vector3(1, 1, 1);
 			public static readonly Vector3 Vector3Forward = new Vector3(0, 0, 1);
-
+				
 			public static Vector3 Vector3Right = new Vector3(1, 0, 0);
 		}
+
+		/// <summary>
+		/// Store common style asset prefixes and suffixes to avoid string spelling errors in code.
+		/// </summary>
+		public static class StyleAssetNames
+		{
+			public const string ALTAS_SUFFIX = "AtlasInfo";
+			public const string PALETTE_SUFFIX = "Palette";
+
+			public const string TOP_MATERIAL_SUFFIX = "TopMaterial";
+			public const string SIDE_MATERIAL_SUFFIX = "SideMaterial";
+		}
+
+		public static class GUI
+		{
+			public static class Colors
+			{
+				public static readonly Color EDITOR_TEXT_COLOR = new Color(0.7f, 0.7f, 0.7f);
+				public static readonly Color EDITOR_NOTE_COLOR = new Color(1.0f, 0.8f, 0.0f);
+			}
+
+			public static class Styles
+			{
+				public static readonly GUIStyle EDITOR_NOTE_STYLE = new GUIStyle { wordWrap = true, normal = { textColor = Colors.EDITOR_NOTE_COLOR } };
+				public static readonly GUIStyle EDITOR_TEXTURE_STYLE_DESCRIPTION_STYLE = new GUIStyle { wordWrap = true, normal = { textColor = Colors.EDITOR_TEXT_COLOR } };
+				public static readonly GUIStyle EDITOR_TEXTURE_THUMBNAIL_STYLE = new GUIStyle { imagePosition = ImagePosition.ImageOnly, wordWrap = true };
+			}
+
+			public static class StyleLabels
+			{
+				public static Dictionary<StyleTypes, string> labels = new Dictionary<StyleTypes, string>()
```

Comment:
Any reason why these descriptions can't be added as `Description` attribute to the enums themselves?
Would avoid updating in multiple files if enums are added/deleted.